packages/plugin-lighthouse/src/lib/utils.unit.test.ts

```diff
@@ -54,3 +59,265 @@ describe('validateOnlyAudits', () => {
     ).toThrow(new AuditsNotImplementedError(['d']));
   });
 });
+
+describe('toAuditOutputs', () => {
+  it('should parse valid lhr details', () => {
+    expect(
+      toAuditOutputs({
+        audit: {
+          scoreDisplayMode: 'binary',
+          title: 'audit',
+          description: 'audit',
+          score: null,
+          id: 'audit',
+        },
+      }),
+    ).toStrictEqual([
+      {
+        displayValue: undefined,
+        score: 1,
+        slug: 'audit',
+        value: 0,
+      },
+    ]);
+  });
+
+  it('should not parse lhr details debugdata', () => {
+    const outputs = toAuditOutputs({
+      audit: {
+        scoreDisplayMode: 'binary',
+        title: 'audit',
+        description: 'audit',
+        score: null,
+        id: 'audit',
+        details: {
+          type: 'debugdata',
+          items: [
+            {
+              cumulativeLayoutShiftMainFrame: 0.000_350_978_852_728_593_95,
+            },
+          ],
+        },
+      },
+    });
+    expect(outputs[0]?.details?.issues[0]).toEqual(
+      expect.objectContaining({
+        message: 'Parsing details from type debugdata is not implemented.',
+        severity: 'info',
+      }),
+    );
+  });
+
+  it('should not parse lhr details filmstrip', () => {
+    const outputs = toAuditOutputs({
+      audit: {
+        scoreDisplayMode: 'binary',
+        title: 'audit',
+        description: 'audit',
+        score: null,
+        id: 'audit',
+        details: {
+          type: 'filmstrip',
+          scale: 3000,
+          items: [
+            {
+              timing: 375,
+              timestamp: 106_245_424_545,
+              data: 'data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wBDAAYEBQYFBAYGBQYHBwY...',
+            },
+          ],
+        },
+      },
+    });
+
+    expect(outputs[0]?.details?.issues[0]).toEqual(
+      expect.objectContaining({
+        message: 'Parsing details from type filmstrip is not implemented.',
+        severity: 'info',
+      }),
+    );
```

Comment:
Instead of accessing possibly undefined properties, you can check relevant areas of the whole object using `objectContaining` as well.

```suggestion
    expect(outputs).toStrictEqual([
      expect.objectContaining({
        details: {
          issues: [
            {
              message:
                'Parsing details from type criticalrequestchain is not implemented.',
              severity: 'info',
            },
          ],
        },
      }),
    ]);
```